cfpb/cfgov-refresh
# PfC: Dynamic Disclosures layout fix

Restores removed PfC LESS files to fix broken layout on Dynamic Disclosures.

## Additions

- Restored LESS file imports in paying for college app's `main.less` file



## How to test this PR

1. Check out branch
2. Run `yarn gulp styles`
3. Navigate to [Dynamic Disclosures](https://www.consumerfinance.gov/paying-for-college2/understanding-your-financial-aid-offer/offer/?iped=154022&pid=BAHCA&totl=70950&tuit=14160&hous=7109&book=1150&tran=500&othr=4743&pelg=0&schg=&stag=&othg=&mta=&gib=&wkst=0&ppl=&parl=&perl=&subl=3500&unsl=6000&gpl=&prvl=&prvi=&prvf=&insl=&insi=&inst=&oid=4D0B9265B45716DF4B81EEB4BE117E1DF69871AF) on production and verify layout is broken
4. Navigate to [Dynamic Disclosures locally](http://localhost:8000/paying-for-college2/understanding-your-financial-aid-offer/offer/?iped=154022&pid=BAHCA&totl=70950&tuit=14160&hous=7109&book=1150&tran=500&othr=4743&pelg=0&schg=&stag=&othg=&mta=&gib=&wkst=0&ppl=&parl=&perl=&subl=3500&unsl=6000&gpl=&prvl=&prvi=&prvf=&insl=&insi=&inst=&oid=4D0B9265B45716DF4B81EEB4BE117E1DF69871AF) and verify that layout is fixed
